### PR TITLE
Fix Maintain Webring Script

### PR DESCRIPTION
**Pull request by iUnstable0**
Fixed

### CODE REVIEW

**Comment by iUnstable0** [comment]
Also please allow workflows to write repository or else this wont work

**Comment by iUnstable0** [comment]
And please enable the workflow i think its disabled manually by someone cuz last time it ran was 2 weeks ago

File: maintain.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+import { readFileSync, writeFileSync } from "fs"
+
+const members = JSON.parse(readFileSync("members.json", { encoding: "utf8" }))
+
+let activeMembers = []
+
+const webringLink = "https://webring.hackclub.com"
+
+async function checkWithBrowser(url, webringLink) {
```
Comment (grymmy) [inline maintain.js:13]: nitpick - would be nice to add a comment to this function describing what it actually does

**Review by grymmy — APPROVED** [review]

**Comment by iUnstable0** [comment]
-.-

**Comment by iUnstable0** [comment]
yeah the branch is main not master

**Comment by grymmy** [comment]
> yeah the branch is main not master

Makes sense, I'll send a pr to try to fix

**Pull request merged**